MC-42708: Scheduled Export does not triggered by cron (#1418)
magento/merchdocs: src/system/data-scheduled-import-export.md

1. When the record is saved, the job appears in the _Scheduled Import/Export_ grid.

   {:.bs-callout-info}

>    Scheduled import/export makes changes to Magento system configuration. After saving, make sure you follow the cache invalidation notice that will appear on top of the Magento admin page and flush the cache in order to apply the new or updated schedule.

Reviewer: ```suggestion
   Scheduled import/export makes changes to the system configuration. After saving, make sure you address the cache invalidation notice that appears at the top of the Admin page and flush the cache in order to apply the new or updated schedule.
```
